templates/patWaitingRoom.html

```diff
@@ -21,12 +21,12 @@
     <script defer src="/__/firebase/init.js"></script> 
 
 
-    <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-  <link rel="stylesheet" type="text/css" href="patWaitingRoom.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../FirebaseRTC/public/prWaitingRoom.html">
```

Comment:
All of the refactoring looks good, however we need to make sure that the files all end up under the public directory and we no longer reference file pathways with ".." or "../.." etc. Check out this link: https://github.com/mellychat/webapp/projects

For the ideal file setup and structure.

Reply:
Lines 25 - 29, 153, 155, 158-159, 162-169 have this issue.